src/Umbraco.Web/Trees/ContentTreeControllerBase.cs

```diff
@@ -18,11 +18,23 @@
 using Umbraco.Web.Actions;
 using Umbraco.Web.Composing;
 using Umbraco.Core.Security;
+using Umbraco.Core.Cache;
+using Umbraco.Core.Configuration;
+using Umbraco.Core.Persistence;
 
 namespace Umbraco.Web.Trees
 {
     public abstract class ContentTreeControllerBase : TreeController
     {
+
+        protected ContentTreeControllerBase(IGlobalSettings globalSettings, IUmbracoContextAccessor umbracoContextAccessor, ISqlContext sqlContext, ServiceContext services, AppCaches appCaches, IProfilingLogger logger, IRuntimeState runtimeState, UmbracoHelper umbracoHelper) : base(globalSettings, umbracoContextAccessor, sqlContext, services, appCaches, logger, runtimeState, umbracoHelper)
+        {
+        }
+
+        protected ContentTreeControllerBase()
+        {
+        }
+
```

Comment:
What's the reason for this empty constructor? And shouldn't it otherwise call the base class?

Reply:
preventing breaking changes in case people are inheriting from this controller.